pypeit/core/telluric.py

```diff
@@ -2263,8 +2447,23 @@ def __init__(self, wave, flux, ivar, gpm, telgridfile, obj_params, init_obj_mode
                 wave, flux, ivar, gpm)
         # 3) Read the telluric grid and initalize associated parameters
         wv_gpm = self.wave_in_arr > 1.0
```

Comment:
Use upper to convert the teltype string to upper, i.e. if upper(teltype) == 'PCA'

Reply:
Shouldn't this be handled in the `pypeitpar`?